packages/driver/src/util/what-is-circular.ts

```diff
@@ -0,0 +1,26 @@
+const lodash = require('lodash')
```

Comment:
Previous file here: https://github.com/cypress-io/what-is-circular/blob/master/index.js